From fe87e18dd1434ffaf61fba70ed3678ccc8b59968 Mon Sep 17 00:00:00 2001
From: Xu Jianqun <jay.xu@rock-chips.com>
Date: Fri, 22 Jan 2016 16:28:35 +0800
Subject: [PATCH] ARM64: dts: rockchip: add clock assign for rk3368 cru

Add "assigned-clocks" for rk3368 cru node, to intalize
clock rate for plls, bus and peripher.

Change-Id: Ic36401fef73b005d778b8ccc8527633af408985c
Signed-off-by: Xu Jianqun <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index e2868db00ec9..41eddb44c630 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -736,6 +736,18 @@
 		rockchip,grf = <&grf>;
 		#clock-cells = <1>;
 		#reset-cells = <1>;
+		assigned-clocks =
+			<&cru PLL_GPLL>, <&cru PLL_CPLL>,
+			<&cru PLL_NPLL>,
+			<&cru ACLK_BUS>, <&cru ACLK_PERI>,
+			<&cru HCLK_BUS>, <&cru HCLK_PERI>,
+			<&cru PCLK_BUS>, <&cru PCLK_PERI>;
+		assigned-clock-rates =
+			<576000000>, <400000000>,
+			<1188000000>,
+			<300000000>, <300000000>,
+			<150000000>, <150000000>,
+			<75000000>, <75000000>;
 	};
 
 	grf: syscon@ff770000 {
-- 
2.35.3

